lib/statesman/adapters/active_record.rb

```diff
@@ -125,6 +120,18 @@ def transition_conflict_error?(e)
         e.message.include?(@transition_class.table_name) &&
           (e.message.include?("sort_key") || e.message.include?("most_recent"))
       end
+
+      def updated_timestamp_options
```

Comment:
Rather than a method which returns a hash, and then merging the hash, why don't you just do something like:

```ruby
def with_timestamp(params = {})
  return params if @transition_class.updated_timestamp_column.nil?
end
```

This prevents a merge, which is slightly faster, and might be better style (but I'm not overly fussed).